OTAnalytics/plugin_ui/toplevel_sections.py

```diff
@@ -16,7 +18,18 @@ def __init__(
     ) -> None:
         super().__init__(**kwargs)
         self.title(title)
-        self.input_values: dict = {ID: ""} if input_values is None else input_values
+        # TODO: Get default values elsewhere!
+        self.input_values: dict = (
+            {
+                ID: "",
+                RELATIVE_OFFSET_COORDINATES: {
+                    EventType.SECTION_ENTER.serialize(): {"x": 0.5, "y": 0.5},
+                    "section-leave": {"x": 0.5, "y": 0.5},
```

Comment:
@briemla do we have a global default defined elsewhere, which we can import here?

Reply:
Defined in adapter_ui.default_values